Dockerfile

```diff
@@ -85,7 +85,8 @@ RUN apk add --update --no-cache tzdata && \
     echo "Europe/London" > /etc/timezone
 
 # libpq: required to run postgres
-RUN apk add --no-cache libpq postgresql-client
+# git: used by the app to report its version to CFE
+RUN apk add --no-cache libpq postgresql-client git
```

Comment:
1. Have you tested this works in a Docker-based environment? I'm surprised git works inside the Docker image, because `.git` is in the .dockerignore, so that dir shouldn't get copied to the Docker image, so git shouldn't work at all.
2. It's not ideal for us to add git to CCQ's attack surface (if someone got inside. Although you've already got chromium 🤯)
3. It's not ideal to run git every time the app starts up

It would be better to do that task at build time i.e. in the Dockerfile. You could add a line just before `43 COPY . .` something like:
```
git rev-parse --short HEAD > ./VERSION
```
and in your app read the file.

Reply:
🤦 I think I got a bit demob happy on my lat day before a break and rushed this. I'll fix it up when I'm back, or if you want to repurpose EL-933 as a 'fix the implementation' ticket and stick it in the backlog, someone else might be able to get to it next week.